src/lib/network_pool/transaction_pool.ml

```diff
@@ -56,7 +56,7 @@ module Diff_versioned = struct
      the checks) and [set_from_gossip_exn] (which just does the mutating the pool),
      and do the same for snapp commands as well.
   *)
-  type t = User_command.t list [@@deriving sexp, yojson]
+  type t = User_command.Stable.Latest.t list [@@deriving sexp, yojson]
```

Comment:
Logging format change

Reply:
Used only in the following code:

```ocaml
[%log' debug t.logger]
              "Failed to batch verify $transaction_pool_diff"
              ~metadata:
                [ ("transaction_pool_diff", Diff_versioned.to_yojson diff) ] ;
```

And in:

```ocaml
[%log trace]
               "Transactions have been applied successfully and is propagated \
                throughout the 'network'"
               ~metadata:
                 [ ( "transactions"
                   , Transaction_pool.Diff_versioned.to_yojson transactions )
                 ] ;
```

It's fine to change the format, as it's mostly for debugging.